app/controllers/accounts/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Accounts::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # You should configure your model like this:
+  # devise :omniauthable, omniauth_providers: [:twitter]
+
+  # You should also create an action method in this controller like this:
+  # def twitter
+  # end
+
+  # More info at:
+  # https://github.com/heartcombo/devise#omniauth
+
+  # GET|POST /resource/auth/twitter
+  # def passthru
+  #   super
+  # end
+
+  # GET|POST /users/auth/twitter/callback
+  # def failure
+  #   super
+  # end
+
+  # protected
+
+  # The path used when OmniAuth fails
+  # def after_omniauth_failure_path_for(scope)
+  #   super(scope)
+  # end
+end
```

Comment:
What is this file doing?

Reply:
**These controllers will be used during the signUp flow , you will come along the code in the up coming PR**

All the files under controller/accounts/ are generated by devise , these files are provided in order to add any extra functionality to devise SignIn/SignUp Flow. These files stays commented unless we want to add any user desired functionality in these controllers as they are inherited from the Devise Controllers.
These controllers were generated using
rails generate devise:controllers accounts